test/utils.js

```diff
@@ -36,11 +36,28 @@ export function almostEqual(a, b, criteria) {
   }
 }
 
-export function checkValue(
-    output, expected, criteria = opFp32AccuracyCriteria) {
+export function compareUlp(a, b, nulp = 1, base = 64) {
```

Comment:
Please split this function into `getBitwise` and `cmopareUlp` as Chai's C++ example.

For `getBitwise`, please change base to a string type, like `float32` and `float64`. BTW, I don't think we support `float64` in spec so far, do we?